### PR TITLE
Use glossary table

### PR DESCRIPTION
**Pull request by mruwnik**
https://github.com/StampyAI/stampy-ui/issues/272
Uses the new [glossary table](https://coda.io/d/AI-Safety-Info_dfau7sl2hmG/Glossary_suZVl#Glossary_tujmw)

### CODE REVIEW

File: app/routes/[sitemap.xml].tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import {LoaderArgs} from '@remix-run/cloudflare'
-import {loadAllQuestions, QuestionStatus, Question} from '~/server-utils/stampy'
+import {loadAllQuestions, QuestionStatus, Question, QuestionState} from '~/server-utils/stampy'
 
 export const loader = async ({request}: LoaderArgs) => {
   const origin = new URL(request.url).origin
   const formatQuestion = ({pageid, updatedAt}: Question) => `
     <url>
-      <loc>${origin}/?state=${pageid}</loc>
+      <loc>${origin}/?state=${pageid}${QuestionState.OPEN}</loc>
```
Comment (mruwnik) [inline app/routes/[sitemap.xml].tsx:8]: this has nothing to do with the glossary, but was needed to be done

**Review by Aprillion — CHANGES_REQUESTED** [review]

File: app/server-utils/stampy.ts [hunk 2]
```diff
@@ -96,6 +96,12 @@ type CodaRow = {
     'Internal?': boolean
     'Questions tagged with this': Entity[]
     'Main question': string | Entity | null
+
+    // Glossary entries
+    definition: string
+    phrase: string
+    aliases: string
+    'question ID': string
```
[the remark this answers is not in the document]
Comment (mruwnik) [reply app/server-utils/stampy.ts:104]: UIID. The aliases can be empty - I'll update that bit

[the remark this answers is not in the document]
Comment (Aprillion) [reply app/server-utils/stampy.ts:104]: I mean, why a new field and why not using the existing "UI ID" field if it's the same table shape type?

File: app/routes/questions/$question.tsx [hunk 3]
```diff
@@ -270,10 +270,12 @@ function Contents({pageid, html, glossary}: {pageid: PageId; html: string; gloss
        */
       fragment.querySelectorAll('.glossary-entry').forEach((e) => {
         const entry = glossaryEntry(e)
+        console.log(entry)
```
Comment (Aprillion) [inline app/routes/questions/$question.tsx:273]: forgotten log?

**Review by Aprillion — COMMENTED** [review]
looks better now 👍 

discriminating union with runtime assertions would be even better to make sure we get what we expect instead of just trusting Coda and doing type assertions, but that can be improved later

**Review by Aprillion — APPROVED** [review]

**Pull request merged**